### PR TITLE
Optionally configure markus to log fixes 402

### PR DESCRIPTION
**Pull request by peterbe**
[See issue](https://github.com/mozilla-services/buildhub/issues/402)

### CODE REVIEW

File: docs/dev.rst [hunk 1]
```diff
@@ -313,3 +313,17 @@ Finally, exit the interactive Docker shell and try to rebuild:
...
     $ docker-compose build buildhub
+
+Debugging metrics logging
+=========================
+
+By default, all metrics logging goes to
+``markus.backends.datadog.DatadogMetrics``. This requires that you have a
+``statsd`` server running on ``$STATSD_HOST:STATSD_PORT``. If you don't
+have that locally you can change it to plain Python logging by setting
+``LOG_METRICS=true``. For example:
```
Comment (leplatrem) [inline docs/dev.rst]: Maybe it would be more explicit to have a string value, like `LOG_METRICS=python` or `LOG_METRICS=datadog` (default)

Comment (peterbe) [reply docs/dev.rst]: Actually, I think that's great idea. I'll switch to that.

Comment (peterbe) [reply docs/dev.rst]: What should happen if someone accidentally sets `LOG_METRICS=logggging`? I.e. a typo.

Comment (leplatrem) [reply docs/dev.rst]: Yes, it's fine, even a `ValueError`

**Pull request merged**